image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -302,7 +298,6 @@ The following block checks for the validity of the provided init bundle. (`helm
   {{ $imagePullSecretNames = append $imagePullSecretNames "stackrox-scanner" }}
 {{ end }}
 
-[<- if and .FeatureFlags.ROX_SCANNER_V4_SUPPORT (not .KubectlOutput) >]
```

Comment:
`not .KubectlOutput` dropped on purpose? If yes, the comment at line :328 below might need to be corrected.